client/config/webpack.dev.js

```diff
@@ -25,9 +26,9 @@ module.exports = merge(common("development"), {
       // In dev mode, populate window._env at build time
       filename: "index.html",
       template: path.resolve(__dirname, "../public/index.html.ejs"),
-      favicon: path.resolve(__dirname, "../public/favicon.ico"),
```

Comment:
It works, so take this or leave it, but I wonder if instead of having `<link rel="icon">` in the EJS file at all we should just have the logic here?
```ts
favicon: path.resolve(__dirname, `../public/${brandType || 'konveyor'}-favicon.ico`
```
that would result in the right `<link>` getting added to the HTML I think. Currently webpack duplicates the `<link rel="icon" href="favicon.ico" />` twice anyway. I think that would mean we don't have to bother with copying the favicon file in the CopyPlugin config above?

We could move this to `client/config/webpack.common.js` so we don't have to repeat it for dev and prod, as well.